visallo/visallo
# Fix geolocation property search filter

Testing Instructions: 
1. Execute a search with a WITHIN geolocation filter.
1. The filter field should appear valid (no red) and it should filter the results.


no CHANGELOG
